script.colorbox/default.py

```diff
@@ -30,6 +30,15 @@ def __init__(self):
             self.image_now = xbmc.getInfoLabel("Player.Art(thumb)")
             self.image_now_fa = xbmc.getInfoLabel("MusicPlayer.Property(Fanart_Image)")
             self.image_now_cfa = xbmc.getInfoLabel("ListItem.Art(fanart)")
+            self.image_now_cpa = xbmc.getInfoLabel("ListItem.Art(poster)")
+            if not HOME.getProperty("cpa_ignore_set") and self.image_now_cpa != self.image_prev_cpa:
+                try:
```

Comment:
so this script creates images for ANY focused item, but doesnt do any garbage collection?